web-marketplace/src/pages/Projects/AllProjects/AllProjects.ProjectFilterMobile.tsx

```diff
@@ -78,6 +82,7 @@ const ProjectFilterMobile = ({
           buyingOptionsFilterOptions={buyingOptionsFilterOptions}
           onCloseFilterModal={onClose}
           mobile
+          classNames="pl-[43px] pr-[43px] overflow-auto h-full"
```

Comment:
seems like this is too much horizontal padding 
from the design https://www.figma.com/design/OXkWyviKwKCDT80HjzCbpE/Project-page-and-project-search-page?node-id=3027-181240&t=mdO1stsgtKS6vhpD-0 it should be 20px